bindings/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+option(FRAMEWORK_USES_PYTHON "Do you want to create the Python bindings" FALSE)
+
+if(FRAMEWORK_USES_PYTHON)
+    find_package(Python3 COMPONENTS Interpreter QUIET)
+    if (Python3_FOUND)
+        execute_process(COMMAND ${Python3_EXECUTABLE}
+            -c "from distutils import sysconfig; print(sysconfig.get_python_lib(1,0,prefix=''))"
+            OUTPUT_VARIABLE _PYTHON_INSTDIR)
+        string(STRIP ${_PYTHON_INSTDIR} FRAMEWORK_PYTHON_INSTALL_DIR)
+        # # Bindings are installed in `BipedalLocomotion` subdirectory.
+        # list(APPEND FRAMEWORK_BINARY_DIRS "${FRAMEWORK_PYTHON_INSTALL_DIR}/BipedalLocomotion")
+    endif()
```

Comment:
Just checking. I doubt you have a virtualenv enabled in your system. Does this installs the files into `/usr` or `/usr/local`?